lxd/device/nic_bridged.go

```diff
@@ -54,7 +54,7 @@ func (d *nicBridged) CanHotPlug() bool {
 
 // CanMigrate returns whether the device can be migrated to any other cluster member.
 func (d *nicBridged) CanMigrate() bool {
-	return d.config["network"] != ""
+	return d.config["network"] != "" || d.config["parent"] != ""
```

Comment:
Hrm, no its not that simple, sadly.

The `parent` could be an unmanaged bridge that we dont know is available on any other system (so "cant migrate").
We need to check if the `parent` is the name of a managed bridge.

However this should be easy because validation does this for various reasons and so we should be able to leverage that.

So we should be able to check if d.network != nil and if its non-nil then return true.

See https://github.com/canonical/lxd/blob/main/lxd/device/nic_bridged.go#L251

Reply:
Please test my assumption here though :)

Reply:
I've tested it and it works like a charm 😊